engine/class_modules/sc_druid.cpp

```diff
@@ -8579,15 +8570,14 @@ struct convoke_the_spirits_t : public druid_spell_t
                    };
 
     cast_list.insert( cast_list.end(),
-                      static_cast<size_t>( rng().range( guidance ? 2.5 : 4, guidance ? 7.5 : 9 ) ),
+                      rng().range<size_t>( guidance ? 2.5 : 4, guidance ? 7.5 : 9 ),
```

Comment:
This results in the 2.5 and 7.5 being truncated to 2 and 7, which changes the behavior of this code.

Reply:
Yes, I mentioned that and a potential solution here: https://github.com/simulationcraft/simc/pull/8652#issuecomment-1948313138